src/index.html

```diff
@@ -8,5 +8,38 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--start-day calendar--month-length">
```

Comment:
How shoud I set moth length?
```suggestion
    <div class="calendar calendar--start-day calendar--month-length--28">
```